pkg/autoscaler/config/config.go

```diff
@@ -132,16 +148,24 @@ func validate(lc *autoscalerconfig.Config) (*autoscalerconfig.Config, error) {
 		return nil, fmt.Errorf("target-burst-capacity must be either non-negative or -1 (for unlimited), was: %f", lc.TargetBurstCapacity)
 	}
 
-	if lc.ContainerConcurrencyTargetFraction <= 0 || lc.ContainerConcurrencyTargetFraction > 1 {
-		return nil, fmt.Errorf("container-concurrency-target-percentage = %f is outside of valid range of (0, 100]", lc.ContainerConcurrencyTargetFraction)
+	if lc.ContainerConcurrencyTargetFraction < 0 || lc.ContainerConcurrencyTargetFraction > 1 {
+		return nil, fmt.Errorf("container-concurrency-target-percentage = %f is outside of valid range of [0, 100]", lc.ContainerConcurrencyTargetFraction)
 	}
 
-	if x := lc.ContainerConcurrencyTargetFraction * lc.ContainerConcurrencyTargetDefault; x < autoscaling.TargetMin {
+	if lc.TargetUtilization <= 0 || lc.TargetUtilization > 1 {
+		return nil, fmt.Errorf("target-utilization = %f is outside of valid range of (0, 100]", lc.TargetUtilization)
+	}
```

Comment:
```suggestion
	if lc.TargetUtilization <= 0 || lc.TargetUtilization > 1 {
		return nil, fmt.Errorf("target-utilization = %f is outside of valid range of (0, 100]", lc.TargetUtilization * 100)
	}
```

Reply:
Done, thanks for your clear explanation.